### PR TITLE
Update endpoints

### LINKED ISSUE #1592: Update endpoints
### Description

Update the following endpoints:
```
REST: GET /reward/inflation to /reward/annual-inflation
WS-RPC: get.reward.inflation to get.reward.annual-inflation
```

```
REST: GET /peers to /network/peers
WS-RPC: get.peers to get.network.peers
```

### Acceptance Criteria
- The above endpoints are updated
- Swagger is up-to-date
- Relevant integration tests are up-to-date

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #1592 

### How was it solved?

- [x] The above endpoints are updated:
  - [x] REST: GET /reward/inflation to /reward/annual-inflation
  - [x] WS-RPC: get.reward.inflation to get.reward.annual-inflation
  - [x] REST: GET /peers to /network/peers
  - [x] WS-RPC: get.peers to get.network.peers
- [x] Code refactoring 
- [x] Swagger is up-to-date
- [x] Relevant integration tests are up-to-date

### How was it tested?
Local

### CODE REVIEW

File: services/blockchain-indexer/shared/dataService/network.js [hunk 1]
```diff
@@ -33,6 +33,90 @@ const getNetworkStatus = async () => {
 	};
 };
 
+const getNetworkPeers = async params => {
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/dataService/network.js:36]: Can we please move the business logic to the business layer?

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Everything looks fine apart from the following:

File: services/blockchain-indexer/shared/dataService/network.js (hunk 1 above)
Comment (nagdahimanshu) [reply services/blockchain-indexer/shared/dataService/network.js]: Done!

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: services/blockchain-indexer/shared/dataService/business/network.js [hunk 2]
```diff
@@ -13,21 +13,37 @@
...
+const { getAvailableModuleCommands, getRegisteredModules } = require('../../constants');
 
-const business = require('./business');
+const getNetworkStatus = async () => {
+	const status = await requestConnector('getNetworkStatus');
 
-const getPeers = async params => {
+	status.moduleCommands = await getAvailableModuleCommands();
+	status.registeredModules = await getRegisteredModules();
+	status.constants = { chainID: status.chainID };
```
Comment (priojeetpriyom) [inline services/blockchain-indexer/shared/dataService/business/network.js:24]: Won't `chainID` be present twice now? Once as `status.chainID` and another as `status.constants.chainID`?

File: services/blockchain-indexer/shared/dataService/business/network.js [hunk 3]
```diff
@@ -79,24 +95,28 @@ const getPeers = async params => {
 	};
 };
 
-const getConnectedPeers = async params => {
-	const response = await getPeers(Object.assign(params, { state: 'connected' }));
+const getNetworkConnectedPeers = async params => {
+	const response = await getNetworkPeers(Object.assign(params, { state: 'connected' }));
```
Comment (priojeetpriyom) [inline services/blockchain-indexer/shared/dataService/business/network.js:99]: maybe we can define a `PEER_STATE` constant like below?

```
PEER_STATE = {
CONNECTED: 'connected',
DISCONNECTED: 'disconnected',
}
```

File: services/blockchain-indexer/methods/dataService/network.js [hunk 4]
```diff
@@ -29,4 +32,27 @@ module.exports = [
...
+		name: 'network.peers',
+		controller: getNetworkPeers,
+		params: {
+			ip: { optional: true, type: 'any' },
+			networkVersion: { optional: true, type: 'any' },
+			state: { optional: true, type: 'any' },
+			height: { optional: true, type: 'any' },
+			limit: { optional: true, type: 'any' },
+			offset: { optional: true, type: 'any' },
+			sort: { optional: true, type: 'any' },
```
Comment (priojeetpriyom) [inline services/blockchain-indexer/methods/dataService/network.js:45]: Can we set a stricter type like `string`, `number`?

**Review by priojeetpriyom — APPROVED** [review]

File: services/blockchain-indexer/methods/dataService/network.js (hunk 4 above)
Comment (nagdahimanshu) [reply services/blockchain-indexer/methods/dataService/network.js]: Done!

File: services/blockchain-indexer/shared/dataService/business/network.js (hunk 2 above)
Comment (nagdahimanshu) [reply services/blockchain-indexer/shared/dataService/business/network.js:24]: @priojeetpriyom Yes, may be this line is unnecessary but i would recommend not to remove now. We can check and remove it later after beta.

File: services/blockchain-indexer/shared/dataService/business/network.js (hunk 3 above)
Comment (nagdahimanshu) [reply services/blockchain-indexer/shared/dataService/business/network.js]: Done!

File: services/blockchain-indexer/shared/dataService/business/network.js (hunk 2 above)
Comment (priojeetpriyom) [reply services/blockchain-indexer/shared/dataService/business/network.js:29]: Can we please create or mention in some exiting card just to keep it noted?

**Review by priojeetpriyom — APPROVED** [review]

**Pull request merged**